nf_core/pipeline-template/{{cookiecutter.name_noslash}}/modules/nf-core/multiqc.nf

```diff
@@ -46,6 +30,7 @@ process MULTIQC {
     custom_config_file = params.multiqc_config ? "--config $mqc_custom_config" : ''
     // TODO nf-core: Specify which MultiQC modules to use with -m for a faster run time
     """
-    multiqc -f $rtitle $rfilename $custom_config_file .
+    echo '$workflow_summary' > workflow_summary_mqc.yaml
+    multiqc -f $rtitle $rfilename $custom_config_file -m fastqc -m custom_content .
```

Comment:
```suggestion
    multiqc -f $rtitle $rfilename $custom_config_file .
```

Reply:
Better to put the modules in `multiqc_config.yaml` e.g. [here](https://github.com/nf-core/viralrecon/blob/75a2f9763e9b4c1aa20ad342ae3ce148c33cc65a/assets/multiqc_config.yaml#L8-L22)